indico/modules/events/abstracts/__init__.py

```diff
@@ -118,3 +118,6 @@ def _is_visible(conf):
                         position=9, visible=lambda event: event.has_feature('abstracts'), static_site=True)
     yield MenuEntryData(title=_("My Abstracts"), name='user_abstracts', visible=_is_visible,
                         endpoint='abstracts.my_abstracts', position=0, parent='call_for_abstracts')
+    yield MenuEntryData(title=_("Submit Abstract"), name='abstract_submission',
+                        visible=lambda event: event.has_feature('abstracts'),
```

Comment:
I think the second approach is better, otherwise people might be wondering why they can't find how to submit an abstract.